src/main/resources/db/changelog/changelog.sql

```diff
@@ -261,3 +261,9 @@ ADD UNIQUE KEY `Accounts-StudyId-SynapseUserId-Index` (`studyId`,`synapseUserId`
 -- changeset bridge:11
 
 CREATE INDEX `Accounts-SynapseUserId-Index` ON `Accounts`(`synapseUserId`);
+
+-- changeset bridge:12
+
+ALTER TABLE `AccountRoles`
+MODIFY COLUMN `role` enum('DEVELOPER','RESEARCHER','ADMIN','WORKER','SUPERADMIN') NOT NULL;
```

Comment:
A brief Google search suggests that some SQL implementations as ordinals (ie, the integer index into the enum values): http://www.mysqltutorial.org/mysql-enum/

If we're deleting tests_users and shifting worker over, there's a chance we might be accidentally elevating all worker accounts to superadmin accounts. (Which wouldn't be the end of the world, since all worker accounts are owned by us, but it's probably not desirable.)

Would it be easier to just keep test_users around?

Reply:
Easy enough, I've put it back. I have not verified that it will not break existing worker accounts in my local environment.